container-runtime/kata-containers/vars.yaml

```
@@ -0,0 +1 @@
+VERSION: "{{ .GVISOR_VERSION }}-{{ .BUILD_ARG_TAG }}"
```

```suggestion
VERSION: "{{ .KATA_CONTAINERS_VERSION }}"
```

The `BUILD_ARG_TAG` is unnecessary if the runtime doesn't depend on kernel modules/ something that's tied to specific talos release